### DISCUSSION TITLE
Can't start on M2 Mac Anymore

### DISCUSSION THREAD

**Discussion by elchupacabrinski** [vladmandic/sdnext#1001] [Q&A] [answered]
Just did a clean install on my M2 Mac and getting this error right after loading the checkpoint (process is aborting afterwards):

loc("varianceEps"("(mpsFileLoc): /AppleInternal/Library/BuildRoots/97f6331a-ba75-11ed-a4bc-863efbbaf80d/Library/Caches/com.apple.xbs/Sources/MetalPerformanceShadersGraph/mpsgraph/MetalPerformanceShadersGraph/Core/Files/MPSGraphUtilities.mm":228:0)): error: input types 'tensor<1x1x1xf16>' and 'tensor<1xf32>' are not broadcast compatible
LLVM ERROR: Failed to infer result type(s).

**Comment by elchupacabrinski** [comment]
Update: Can start with --use-cpu all, but is obviously very slow

**Reply by vladmandic** [reply, marked as answer]
that's most likely due to torch 2.0.1 causing problems, i've just locked down required version on mac to 2.0.0.
either downgrade torch manually or run with `--reinstall` flag.

**Reply by elchupacabrinski** [reply]
weird. I tried that yesterday already without effect but now it works. Thanks :D

**Reply by vladmandic** [reply]
because i just locked down version - i said that :)

**Comment by sullenfish** [comment]
@elchupacabrinski - I had some luck this morning with torch nightly. If you're feeling adventurous, add in to your `webui-user.sh`:

`export TORCH_COMMAND="--pre torch torchvision --index-url https://download.pytorch.org/whl/nightly/cpu"`

Of course that comes with all the caveats of it being pre-release, but it gives me slightly better results than torch 2.0.0 on an M1.
